plugins/package-managers/python/src/main/kotlin/Poetry.kt

```diff
@@ -63,7 +65,8 @@ class Poetry(
     override fun transformVersion(output: String) = output.substringAfter("version ").removeSuffix(")")
 
     override fun resolveDependencies(definitionFile: File, labels: Map<String, String>): List<ProjectAnalyzerResult> {
-        val resultsForScopeName = SCOPES_NAMES.associateWith { inspectLockfile(definitionFile, it) }
+        val scopeName = parseScopeNamesFromPyProject(definitionFile.resolveSibling(Pip.PYPROJECT_FILENAME))
```

Comment:
Would [1] be valuable to have in the commit message?

[1] https://python-poetry.org/docs/pyproject/#dependencies-and-dependency-groups